scripts/exporter.py

```diff
@@ -323,6 +324,62 @@ def main(self) -> None:
         raise NotImplementedError("Marching cubes not implemented yet.")
 
 
+@dataclass
+class ExportSDFMarchingCubesMesh(Exporter):
+    """Load SDF checkpoint, run marching cubes, extract mesh."""
+
+    resolution: int = 1024
+    """Marching cube resolution."""
+    simplify_mesh: bool = False
+    """Whether to simplify the mesh."""
+    bounding_box_min: Tuple[float, float, float] = (-1.0, -1.0, -1.0)
+    """Minimum of the bounding box."""
+    bounding_box_max: Tuple[float, float, float] = (1.0, 1.0, 1.0)
+    """Maximum of the bounding box."""
+    px_per_uv_triangle: int = 4
+    """Number of pixels per UV triangle."""
+    unwrap_method: Literal["xatlas", "custom"] = "xatlas"
+    """The method to use for unwrapping the mesh."""
+    num_pixels_per_side: int = 2048
+    """If using xatlas for unwrapping, the pixels per side of the texture image."""
+    target_num_faces: Optional[int] = 50000
+    """Target number of faces for the mesh to texture."""
+
+    def main(self) -> None:
+        """Main function."""
+        if not self.output_dir.exists():
+            self.output_dir.mkdir(parents=True)
+
+        _, pipeline, _ = eval_setup(self.load_config)
+
+        CONSOLE.print("Extract mesh with marching cubes and may take a while")
+
+        assert self.resolution % 512 == 0
+        # for sdf we can multi-scale extraction.
+        get_surface_sliding(
```

Comment:
Can we rename this function? "get" implies you are returning something, but instead you are writing to the `output_path`. Maybe `write_mesh_with_marching_cubes`? You can decide, but something with writing instead of "get"ing would be nicer IMO.

Reply:
Agreed, changed it to more closely follow other examples like the `ExportPoissonMesh`